marklogic-data-hub/src/main/resources/hub-internal-artifacts/step-definitions/mapping/marklogic/default-mapping.step.json

```diff
@@ -14,6 +14,7 @@
     "collections": [
       "default-mapping",
       "mdm-content"
-    ]
+    ],
+    "validateEntity": false
```

Comment:
Is this something that users should be able to set from GUI as well?

Reply:
Eventually. This is just getting the backend in place.